assets/scripts/app/__tests__/BlockingError.test.js

```diff
@@ -18,13 +18,17 @@ function getInitialState (errorType) {
 describe('BlockingError', () => {
   it.each(Object.keys(ERRORS))('renders %s', (error) => {
     const initialState = getInitialState(ERRORS[error])
-    const wrapper = renderWithReduxAndIntl(<BlockingError />, { initialState })
-    expect(wrapper.asFragment()).toMatchSnapshot()
+    const { asFragment } = renderWithReduxAndIntl(<BlockingError />, {
```

Comment:
@burrelle I understand the changes here, but could you briefly describe the justification for the general strategy you have here of using asFragment vs wrapper? It would be helpful to leave a note for future devs. 

I find it confusing since 'asFragment' is part of the testing library API: 
https://testing-library.com/docs/react-testing-library/api#asfragment

But we aren't using that here, just calling this 'asFragment'. Unless I'm missing something fancy going on.